lxd/backup.go

```diff
@@ -554,3 +554,80 @@ func volumeBackupWriteIndex(s *state.State, projectName string, volumeName strin
 
 	return nil
 }
+
+func pruneExpiredStorageVolumeBackupsTask(d *Daemon) (task.Func, task.Schedule) {
+	f := func(ctx context.Context) {
+		s := d.State()
+
+		opRun := func(op *operations.Operation) error {
+			return pruneExpiredStorageVolumeBackups(ctx, s)
+		}
+
+		op, err := operations.OperationCreate(s, "", operations.OperationClassTask, operationtype.BackupsExpire, nil, nil, opRun, nil, nil, nil)
+		if err != nil {
+			logger.Error("Failed creating expired storage volume backups operation", logger.Ctx{"err": err})
+			return
+		}
+
+		logger.Info("Pruning expired storage volume backups")
+		err = op.Start()
+		if err != nil {
+			logger.Error("Failed starting expired storage volume backups operation", logger.Ctx{"err": err})
+			return
+		}
+
+		err = op.Wait(ctx)
+		if err != nil {
+			logger.Error("Failed pruning expired storage volume backups", logger.Ctx{"err": err})
+			return
+		}
+
+		logger.Info("Done pruning expired storage volume backups")
+	}
+
+	f(context.Background())
```

Comment:
`d.ShutdownCtx` maybe?

Reply:
It's executed here once to not wait until `time.Hour` passes until it gets executed for the first time by the task scheduler right?

Reply:
> d.ShutdownCtx maybe?

Good call. I'll fix that.

> It's executed here once to not wait until time.Hour passes until it gets executed for the first time by the task scheduler right?

Correct.